hack/verify-test-featuregates.sh

```diff
@@ -26,7 +26,7 @@ cd "${KUBE_ROOT}"
 rc=0
 
 # find test files accessing the mutable global feature gate or interface
-direct_sets=$(grep -n --include '*_test.go' -R 'MutableFeatureGate' . 2>/dev/null) || true
+direct_sets=$(find . -follow -name '*_test.go' -exec grep -Hn 'MutableFeatureGate' {} \; 2>/dev/null) || true
```

Comment:
We should use `-L` instead of `-follow`, `-follow` is deprecated on linux (GNU) and macOS supports `-L` because this is POSIX.

Reply:
Thanks for comments.
I changed from "-follow" to "-L" and checked to work properly in my local environment.